### PR TITLE
loading indicator when clicking save button

### PR DESCRIPTION
**Pull request by divinity76**
<strike>... should probably find a better place to load the image from. maybe someone on github has an idea?</strike>
edit: bundled the image: edd2b843716214850f761811dd2b929d62101f55

before clicking save: 
image

after clicking save: 
image

### CODE REVIEW

File: web/js/init.js [hunk 1]
```diff
@@ -613,5 +613,12 @@ $(document).ready(function(){
                 $('.l-unit').find('.ch-toggle').prop('checked', false);
                 $('.l-unit.selected').find('.ch-toggle').prop('checked', true);
             });
+            // todo: maybe give the save button id?
+            $("[data-id=vstobjects][data-action=submit]").on("click", function () {
+              let loadingAnimationEle = document.createElement("img");
+              loadingAnimationEle.src = "https://raw.githubusercontent.com/phpmyadmin/phpmyadmin/7a6d4e68d3f6238f837d92c84c80081436ff02cc/themes/original/img/ajax_clock_small.gif";
```
Comment (ScIT-Raphael) [inline web/js/init.js]: We should not use external sources.

**Review by ScIT-Raphael — CHANGES_REQUESTED** [review]

File: web/js/init.js (hunk 1 above)
Comment (divinity76) [reply web/js/init.js]: how about edd2b843716214850f761811dd2b929d62101f55

**Comment by jaapmarcus** [comment]
Screenshot 2022-07-31 at 11 01 15

@divinity76  any issues with it before I merged it?

**Comment by divinity76** [comment]
@jaapmarcus nope
LGTM 👍

**Review by jaapmarcus — APPROVED** [review]

**Review by ScIT-Raphael — APPROVED** [review]

**Pull request merged**